vue/sbc-common-components/src/components/SbcLogin.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-container class="view-container">
+    <v-row justify="center">
+      <v-col class="text-center">
+        <div class="mt-5 mb-10 font-weight-bold">
+          <h1>BC Registries Account Login</h1>
+        </div>
+        <v-card
+          class="mx-auto"
+          max-width="460"
+        >
+          <v-card-title>
+            <v-img
+              max-height="260"
+              src="../assets/img/BCReg_Generic_Login_image.jpg"
+              alt="Generic Login Image">
+            </v-img>
+          </v-card-title>
+          <v-card-text >
+            <v-menu>
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  large
+                  class="white--text font-weight-bold"
+                  aria-label="log in"
+                  id="loginBtn"
+                  v-on="on">
+                  <span>Log in to my BC Registries Account</span>
+                  <v-icon class="mr-n1 ml-2">mdi-menu-down</v-icon>
+                </v-btn>
+              </template>
+              <sbc-auth-menu :fromLogin="true" :redirect-on-login-success="redirectUrl" />
```

Comment:
Does the signin button on the BCROS pages need to specify any of these props when it uses the SbcAuthMenu component?

![image](https://user-images.githubusercontent.com/10002889/150436929-3d3ac4cb-ddde-431b-b67b-e8f001f39732.png)

Reply:
I mean, do I need to specify those props from BCROS pages?

Reply:
You may need to test the bookmark logic. If user bookmark a specify page that need to login (i.e. dashboard), your BCROS page should handle it.

Reply:
Yeah, that's working. Thanks for the feedback.